docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     privileged: true
     volumes:
         - scripts:/usr/src/slapi/scripts
+        - brain:/usr/src/slapi/brain
```

Comment:
Do these directories need to be created on the host before running?  If so should that be in the README for development?